serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -22,116 +24,107 @@ import sbt.serialization.json.{
 sealed trait SerializedValue {
   def parse[T](implicit unpickler: Unpickler[T]): Try[T]
 
-  // TODO - expose toJson, toBinary etc. (anyhting we need).
+  def toJsonString: String
+
+  // TODO toBinary if/when we add binary pickling
+
+  // private because we don't have our chosen
+  // json AST library in the public API
+  private def toJValue: JValue =
+    this match {
+      case jsonValue: JsonValue =>
+        jsonValue.pickledValue.parsedValue
+      case lazyValue: LazyValue[_] =>
+        lazyValue.toJson.pickledValue.parsedValue
+    }
+
+  final override def equals(other: Any): Boolean =
+    other match {
+      case null => false
+      case sv: SerializedValue => toJValue.equals(sv.toJValue)
```

Comment:
This is the only reason we want JValue.  Also, "JValue" does not have a safe equality, so this is probably breaking test code.

Ideally there should be an alternative equals comparison here, or you need to use the `JsonMethods.jvalueEquals`